### PR TITLE
Fix typo in a comment

### PR DESCRIPTION
**Pull request by hezhizhen**
`usualy` -> `usually`.

I ran `./tools/build` and some lines were moved.

### CODE REVIEW

File: examples/custom-errors/custom-errors.go [hunk 1]
```diff
@@ -24,15 +24,13 @@ func (e *argError) Error() string {
 
 func f(arg int) (int, error) {
 	if arg == 42 {
-
```
Comment (eliben) [inline examples/custom-errors/custom-errors.go:27]: Can you revert these changes? It's probably done by your editor (`tools/build` doesn't modify the `.go` input files).

These empty lines help with LHS comment layout

**Review by eliben — COMMENTED** [review]
The typo fix SGTM, but please address comment and then rerun build

File: examples/custom-errors/custom-errors.go (hunk 1 above)
Comment (hezhizhen) [reply examples/custom-errors/custom-errors.go]: Sorry about that. Added back these empty lines. PTAL

**Review by eliben — APPROVED** [review]
